### PR TITLE
Regenerate all resource docs

### LINKED ISSUE #3075: Functions in Go have wrong names
### Problem description

Functions in Go are in the form `GetThing`, but our resource docs show them in the code example as `LookupThing` which is wrong.

image

#### Examples
- https://www.pulumi.com/docs/reference/pkg/gcp/storage/getprojectserviceaccount/#getprojectserviceaccount
- https://www.pulumi.com/docs/reference/pkg/gcp/compute/getnetwork/#using
- https://www.pulumi.com/docs/reference/pkg/aws/ec2/getvpc/#using

### PR DESCRIPTION
**Pull request by praneetloke**
### Proposed changes

The main change in this PR is to fix an issue with the Function names in Go.

We should wait to merge this _after_ #3093, #3098, #3099. I might have to rebase and regenerate after those PRs are merged.

### Unreleased product version (optional)

N/A

### Related issues (optional)

Fixes #3075. The generator fix is in: https://github.com/pulumi/pulumi/pull/4490.

### CODE REVIEW

File: content/docs/reference/pkg/aws/ec2/getvpcendpoint.md [hunk 1]
```diff
@@ -991,7 +991,7 @@ The following output properties are available:
 ## Supporting Types
 
 
-<h4 id="getvpcendpointdnsentry">Get<wbr>Vpc<wbr>Endpoint<wbr>Dns<wbr>Entry</h4>
+<h4 id="getvpcendpointdnsentry">get<wbr>Vpc<wbr>Endpoint<wbr>Dns<wbr>Entry</h4>
```
Comment (justinvp) [inline content/docs/reference/pkg/aws/ec2/getvpcendpoint.md:994]: Shouldn't these remain title case?

Comment (praneetloke) [reply content/docs/reference/pkg/aws/ec2/getvpcendpoint.md:994]: Yeah you are right. Let me look at it. I also need to rebase with `master` in this repo and regen all docs here.

Comment (praneetloke) [reply content/docs/reference/pkg/aws/ec2/getvpcendpoint.md]: The title is fixed now.

**Comment by praneetloke** [comment]
Now that the generator fix is merged in I will wait to merge this as @stack72 mentioned all of the providers will rolled tomorrow anyway, which will trigger a docs PR for each of them anyway. So we may not need this PR after all.

**Pull request closed without merging**